src/averageAges.js

```diff
@@ -14,12 +14,27 @@
  *
  * @return {number}
  */
+
+const findAverage = function(arr, special, people) {
```

Comment:
What doest `special` mean?

Reply:
It is not the best idea to try to combine this logic in this way

Reply:
I guess you are right

Reply:
I`ve redone function findAverage to a simplier for the first two functions, will it be alright?